### PR TITLE
Feat/wizeline branded

### PR DESCRIPTION
**Pull request by MoisesFlores22**
What this Pr does?

Create the wizeline-branded component appear in the bottom of the chat 

pictures:

screen shot 2017-02-15 at 12 15 49 pm

### CODE REVIEW

File: src/WizelineBranded.tsx [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+
+
+class WizelineBranded extends React.Component<{}, {}> {
+
+    render() {
+        return(
+            <div className='wizeline-branded'>
+                <span className='wizeline-span'> 
+                    <p className='wizeline-branded-font'>Powered by</p>
+                    <svg className='wizeline-logo'>
```
Comment (erikvilla) [inline src/WizelineBranded.tsx]: svg should be an asset

File: wz-botchat.css [hunk 2]
```diff
@@ -550,6 +550,32 @@
...
+    }
+
+    .wizeline-span {
+      display: flex;
+    }
+
+    .wizeline-branded-font {
+      font-size: 8px;
+      margin: 2px 4px 0 0 !important;
+
```
Comment (erikvilla) [inline wz-botchat.css]: space

**Comment by MoisesFlores22** [comment]
@erikvilla  the changes are ready

File: src/WizelineBranded.tsx [hunk 3]
```diff
@@ -0,0 +1,15 @@
+import * as React from 'react';
+
+class WizelineBranded extends React.Component<{}, {}> {
+
+    render() {
+        const image = '/static/assets/wizeline-logo-gray.svg';
```
Comment (erikvilla) [inline src/WizelineBranded.tsx]: It's better if you define this outside `render` function so it's not defined on every render call

**Comment by erikvilla** [comment]
Did you test this in mobile?

**Comment by MoisesFlores22** [comment]
yes i test it

**Pull request merged**